src/hyperlight_host/src/metrics/mod.rs

```diff
@@ -132,27 +136,17 @@ mod tests {
         cfg_if::cfg_if! {
             if #[cfg(feature = "function_call_metrics")] {
                 use metrics::Label;
-                // Verify that the histogram metrics are recorded correctly
-                assert_eq!(snapshot.len(), 4, "Expected two metrics in the snapshot");
 
-                // 1. Host print duration
-                let histogram_key = CompositeKey::new(
-                    metrics_util::MetricKind::Histogram,
-                    Key::from_parts(
-                        METRIC_HOST_FUNC_DURATION,
-                        vec![Label::new("function_name", "HostPrint")],
-                    ),
-                );
-                let histogram_value = &snapshot.get(&histogram_key).unwrap().2;
-                assert!(
-                    matches!(
-                        histogram_value,
-                        metrics_util::debugging::DebugValue::Histogram(histogram) if histogram.len() == 1
-                    ),
-                    "Histogram metric does not match expected value"
-                );
+                let expected_num_metrics = if cfg!(all(feature = "seccomp", target_os = "linux")) {
+                    3 // if seccomp enabled, the host call duration metric is emitted on a separate thread which this local recorder doesn't capture
```

Comment:
Why did this change? Wasn't this always the case?

Reply:
We can run this test together with other tests now since it's not ignore anymore